Lombiq.Tests.UI/BasicOrchardFeaturesTesting/OrchardAuditTrailFeatureTestingUITestContextExtensions.cs

```diff
@@ -0,0 +1,50 @@
+using Atata;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Helpers;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+using System.Threading.Tasks;
+
+namespace Lombiq.Tests.UI.BasicOrchardFeaturesTesting;
+
+/// <summary>
+/// Provides a set of extension methods for Orchard Core Audit Trail feature testing.
+/// </summary>
+public static class OrchardAuditTrailFeatureTestingUITestContextExtensions
+{
+    public static Task TestAuditTrailAsync(this UITestContext context) =>
+    context.ExecuteTestAsync(
+        "Test Audit Trail",
+        async () =>
+        {
+            var auditTrailPath = "/AuditTrail";
+            var auditTrailTestPageTitle = "Audit Trail Test Page";
+
+            await context.EnableFeatureDirectlyAsync("OrchardCore.AuditTrail");
+            await context.GoToAdminRelativeUrlAsync("/Settings" + auditTrailPath);
+
+            await context.GoToEditorTabAsync("Content");
+
+            await context.SetCheckboxValueAsync(By.XPath("//input[@value='Page']"));
+
+            await context.ClickReliablyOnSubmitAsync();
+
+            var contentItemsPage = await context.GoToContentItemsPageAsync();
+            context.RefreshCurrentAtataContext();
+            contentItemsPage
+                .CreateNewPage()
+                    .Title.Set(auditTrailTestPageTitle)
+                    .Publish.ClickAndGo()
+                .AlertMessages.Should.Contain(message => message.IsSuccess);
+
+            await context.GoToAdminRelativeUrlAsync(auditTrailPath);
+
+            context.Exists(ByHelper.TextContains("of the Page"));
+
+            context.Exists(ByHelper.TextContains("was published"));
+
+            context.Exists(ByHelper.TextContains("was created"));
```

Comment:
As mentioned previously, these aren't unique to Audit Trail Test Page. Also check for "Audit Trail Test Page" to be included with **these texts here**, not separately. Because otherwise, this doesn't test for the two events of Audit Trail Test Page, rather it checks for three texts that can be satisfied by any random event independent of Audit Trail Test Page, and then checks for any random "Audit Trail Test Page" text. Check these exactly:

![image](https://github.com/Lombiq/UI-Testing-Toolbox/assets/1976647/364fe802-0119-4ded-9151-b81e76ec9bd6)